### PR TITLE
[new release] omd (2.0.0.alpha2)

### PR DESCRIPTION
**Pull request by shonfeder**
A Markdown frontend in pure OCaml

- Project page: <a href="https://github.com/ocaml/omd">https://github.com/ocaml/omd</a>

##### CHANGES:

- Clean up AST representation (ocaml/omd#229, @sonologico)

- Restore table of contents generation functionality (ocaml/omd#240, @sonologico)

- Fix parsing of <...>-style links and inline code parsing (ocaml/omd#233, @SquidDev)

- Fix handing of the nonbreaking whitespace character, ASCII code 160 (ocaml/omd#247,
  @shonfeder)

- Downgrade dune requirements to 2.0 (@nobj)

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The version number isn't quite right. It should be 2.0.0~alpha2 (~ instead of .)

**Comment by shonfeder** [comment]
I think I've fixed the version (the directory name). Sorry about that!

File: packages/omd/omd.2.0.0~alpha2/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+license: "ISC"
+tags: ["org:ocamllabs" "org:mirage"]
+homepage: "https://github.com/ocaml/omd"
+bug-reports: "https://github.com/ocaml/omd/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.04"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/omd/omd.2.0.0~alpha2/opam:27]:
```suggestion
  ["dune" "subst"] {dev}
```

This is fixed only after dune 2.7, I think no extra action needs to be taken on your side

**Comment by mseri** [comment]
Looks good now, thanks

**Comment by shonfeder** [comment]
Thanks for the fix!

**Pull request merged**